src/_nebari/upgrade.py

```diff
@@ -244,6 +320,7 @@ def set_nested_item(config: dict, config_path: list, value: str):
             config[config_path[-1]] = value
 
         def update_image_tag(config, config_path, current_image, new_version):
+            """Update the image tag in the configuration."""
```

Comment:
This is missing the `Args` and `Returns` documentation.

Reply:
I've added some additional docstrings in [`8f83443`](https://github.com/nebari-dev/nebari/pull/2512/commits/8f834431526654cccc52a1ac596d72d1aba9f063), Plus i've also added some typing to the args which had it missing